### ISSUE TITLE
Other way around?

### ISSUE THREAD

**Issue by iroQuai** [philippe44/AirConnect#381]
Is there a way to do this other way around? Using the Chromecast protocol to send audio to airplay devices?

I've searched quite a lot but can't find a tool that's capable of this (at least not the way airconnect offers from airplay to Chromecast)

This would be extremely useful for all Android users with Sonos (or symfonisk) devices (and there re probably a lot more use cases)

Thanks I'm advance for helping out!

**Comment by coolspot18** [comment]
Same here, I'm looking to re-purpose some old Airplay/DLNA speakers (Pioneer A4s) to use Chromecast.

**Comment by philippe44** [comment]
No, it's not possible due to the way CC devices are designed (especially around security)
